src/classlibnative/bcltype/varargsnative.cpp

```diff
@@ -87,8 +87,8 @@ static void InitCommon(VARARGS *data, VASigCookie** cookie)
     //  STACK_GROWS_DOWN_ON_ARGS_WALK
 
     //   <return address>  ;; lower memory                  
-    //   <varargs_cookie>         \
-    //   <argN>                    \
+    //   <varargs_cookie>         '\'
```

Comment:
Hmm, I've tried this in the compiler explorer and it didn't show any warnings for GCC (arm / arm64 / x64) tried version 4.5.4 and 8.2.

Reply:
I get multiline comment warning with 

Thread model: posix
gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.11)

using ubuntu 16.04.

Reply:
Hmm, it should not show the warning unless we compile with `-Wall` or `-Wcomment`. Strange.

Reply:
We are compiling with -Wall. This file is a copy/paste of clang file.

https://github.com/dotnet/coreclr/blob/439193cf9e63678138a74d8020dd98a7096c5d49/src/pal/tools/gcc-compiler-override.txt#L9